applications/mne_scan/plugins/eegosports/eegosportsdriver.h

```diff
@@ -144,7 +145,7 @@ class EEGoSportsDriver
     * @param [in] bMeasureImpedance Flag for measuring impedances.
     */
     bool initDevice(bool bWriteDriverDebugToFile,
-                    QString sOutpuFilePath,
+                    const QString sOutpuFilePath,
```

Comment:
please make use of const references for `QString`: `const QString& sOutpuFilePath`